app/services/discourse_calendar/holiday.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+require "holidays"
+
+module DiscourseCalendar
+  class Holiday
```

Comment:
My thought behind this new holiday service is for it to be an interface and wrapper for the Holidays gem. It will return the results from the gem and also add a new `disabled` field. I also created it so I didn't have to duplicate the disabled functionality - the admin UI needs this functionality so it knows which holidays are enable/disabled, and the `create_holiday_events` job needs it to only add enabled holidays to the calendar. I also think this is where custom holidays would get incorporated at.